providers/msTeams/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@novu/msteams",
```

Comment:
```suggestion
  "name": "@novu/ms-teams",
```

A small suggestion, could we add a "-" in the package names used here?

Reply:
Sure, I'll change it

Reply:
Done